### PR TITLE
Bug 1905977: Gherkin script pipeline metrics odc 3711

### PR DESCRIPTION
**Pull request by sanketpathak**
Epic: https://issues.redhat.com/browse/ODC-3711
Story: https://issues.redhat.com/browse/ODC-4855

### CODE REVIEW

File: frontend/packages/dev-console/integration-tests/features/pipelines/pipeline_metrics_ODC-3711.feature [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    Given pipeline is present 
+    When user clicks on available pipeline
+    Then user is able to see Dashboard tab in pipeline details page
+
+
+@regression
+Scenario: Graphs in dashboard tab
+    Given user is in pipeline details tab
+    And pipeline run is present
+    When user clicks on dashboard tab
```
Comment (karthikjeeyar) [inline frontend/packages/dev-console/integration-tests/features/pipelines/pipeline_metrics_ODC-3711.feature:22]: The title `dashboard` is likely to be changed, not confirmed yet. will you create a follow up PR to change it??

File: frontend/packages/dev-console/integration-tests/features/pipelines/pipeline_metrics_ODC-3711.feature [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    Then user can see Time Range and Refresh Interval fields with value as 1 day and 30 seconds respectively
```
Comment (karthikjeeyar) [inline frontend/packages/dev-console/integration-tests/features/pipelines/pipeline_metrics_ODC-3711.feature:23]:
```suggestion
    Then user can see Time Range and Refresh Interval dropdowns with a default value of 1 day and 30 seconds respectively
```

File: frontend/packages/dev-console/integration-tests/features/pipelines/pipeline_metrics_ODC-3711.feature [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    When user clicks on dashboard tab
+    Then user can see empty page with message "Start your pipeline to view pipeline metrics"
+
+
+@regression
+Scenario: No datapoint graphs in dashboard tab
+    Given user is in pipeline details tab
+    And pipeline run has just started
+    When user clicks on dashboard tab
+    Then user can see Time Range and Refresh Interval fields with value as 1 day and 30 seconds respectively
```
Comment (karthikjeeyar) [inline frontend/packages/dev-console/integration-tests/features/pipelines/pipeline_metrics_ODC-3711.feature:41]: change it based on my  previous comment

File: frontend/packages/dev-console/integration-tests/features/pipelines/pipeline_metrics_ODC-3711.feature (hunk 1 above)
Comment (sanketpathak) [reply frontend/packages/dev-console/integration-tests/features/pipelines/pipeline_metrics_ODC-3711.feature:22]: Sure I'll create a gherkin task for it. Is there a bug/task created for this

Comment (karthikjeeyar) [reply frontend/packages/dev-console/integration-tests/features/pipelines/pipeline_metrics_ODC-3711.feature:22]: not yet, but will notify you once it is created.

**Review by gajanan-more — CHANGES_REQUESTED** [review]
Please update the indentation of script.

**Pull request merged**